skyportal/handlers/api/candidate.py

```diff
@@ -447,6 +447,8 @@ def post(self):
             ]
         )
         DBSession().commit()
+        if not obj_already_exists:
+            obj.add_linked_thumbnails()
```

Comment:
I know this is basically the logic we used to have, but I'm curious: what do we do if this step fails?

Reply:
Well, currently it returns with an error. What do you propose we should do?

Reply:
@stefanv I dont think this should be too volatile - at this point basically it is just adding a single row to the Thumbs table in the DB with a string link to the external thumbs.

Reply:
What I meant, and maybe this is already handled, is that we should attempt again in the future if fetching the URL fails.  So, maybe do this call every time an obj is accessed and the URL is not filled out.

Reply:
Looks like this is the logic below already, maybe.

Reply:
@stefanv yes, exactly. The line above adds the two thumbnail links that we already know without making an external request. PS1 requires making the external request - if that fails, it'll try again the next time the source is accessed